cmake/KVIrc.nsi.cmake

```diff
@@ -16,9 +16,9 @@ Name "@NICENAME@"
 !define URL_UPDATE 'http://www.kvirc.net/'
 !define PUBLISHER 'Szymon Stefanek and The KVIrc Development Team'
 
-; Git release, eg: KVIrc-4.0.0-rc3-git-4423-5ac91f3.exe
+; Git release, e.g. KVIrc-4.3.2-dev-git-5465-ge352609-x86_64.exe
```

Comment:
-x86_64 doesn't belong here. Do you see it added below?
In appveyor config, it's appended to exe name in the very end of build.